src/util/rust_util.rs

```diff
@@ -7,6 +7,31 @@ pub const fn min_of_usize(a: usize, b: usize) -> usize {
     }
 }
 
+// likely() and unlikely() compiler hints in stable Rust
+// [1]: https://github.com/rust-lang/hashbrown/blob/a41bd76de0a53838725b997c6085e024c47a0455/src/raw/mod.rs#L48-L70
+// [2]: https://users.rust-lang.org/t/compiler-hint-for-unlikely-likely-for-if-branches/62102/3
+#[cfg(not(feature = "nightly"))]
+#[inline]
+#[cold]
+fn cold() {}
+
+#[cfg(not(feature = "nightly"))]
```

Comment:
I think you missed these lines:
```rust
#[cfg(feature = "nightly")]
use core::intrinsics::{likely, unlikely};
```
https://github.com/rust-lang/hashbrown/blob/a41bd76de0a53838725b997c6085e024c47a0455/src/raw/mod.rs#L45

Otherwise we cannot compile with a nightly compiler.

Reply:
Yep my bad. Done